src/Azure/Akka.Streams.Azure.StorageQueue.Tests/QueueSinkSpec.cs

```diff
@@ -85,7 +84,9 @@ public async Task A_QueueSink_should_skip_failing_messages_if_supervision_strate
             probe.SendNext("2");
             probe.SendComplete();
             await task;
-            (await Queue.ReceiveMessagesAsync()).Value[0].MessageText.Should().Be("2");
+            var msg = (await Queue.ReceiveMessagesAsync()).Value;
```

Comment:
We should probably add a timeout condition here. We have a convenience method called `ShouldCompleteWithin(TimeSpan)` that wraps a Task with a timeout to signal that an async operation takes too long to complete.